res/values/strings.xml

```diff
@@ -679,6 +679,12 @@
     <string name="qrscan_contains_url">Scanned QR code URL:\n\n%1$s</string>
     <string name="qrscan_fingerprint_label">Fingerprint</string>
     <string name="qrscan_x_verified_introduce_myself">%1$s verified, introduce myself…</string>
+    <string name="withdraw_verifycontact_explain">This QR code can be scanned by others to setup a contact with you.\n\nIf you do not want that any longer, you can withdraw it. You can revive it by scanning it again.</string>
+    <string name="withdraw_verifygroup_explain">This QR code can be scanned by others to join the group \"%1$s\".\n\nIf you do not want that any longer, you can withdraw it. You can revive it by scanning it again.</string>
```

Comment:
why is it necessary to add 'by others' at all? You could also join a group with a second address.

Reply:
Because if we say "This QR code can be scanned to join the group XY", some people will wonder "But I just scanned it, why didn't I join the group?"

There are two types of "scan" here:
- scan your own QR code
- Let others scan your QR code

and we have to make it clear which one we mean every time we write "scan". This was the result of an actual user test with n=1 non-technical participants I made :)

> You could also join a group with a second address.

I think that it will be clear that "others" includes "other addresses of yourself"; in case you don't agree it's probably best to go ask non-technical users again (of course they will have to be technical enough to understand why you would want to join your own group at all, and to understand that you could scan your QR code with another account for this)

Reply:
```suggestion
    <string name="withdraw_verifygroup_explain">This QR code can be scanned to join the \"%1$s\" group.\n\nIt can be withdrawn at any time, and revived by scanning it again.</string>
```